### PR TITLE
Add packer build

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by venkatBala** [comment]
sweet!!

**Comment by wjcunningham7** [comment]
@venkatBala handing this off to you to review/complete when you have time

File: covalent.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    most_recent = true
+    owners      = ["099720109477"]
+  }
+
+  ssh_username = "ubuntu"
+}
+
+build {
```
Comment (venkatBala) [inline covalent.pkr.hcl:64]: would a `tags` block with some metadata about covalent and the python version help?

**Review by venkatBala — DISMISSED** [review]

**Comment by venkatBala** [comment]
updated to this: https://github.com/AgnostiqHQ/terraform/pull/1

**Comment by venkatBala** [comment]
@wjcunningham7 can this one be closed... since all infra related stuff has been moved out of core?

**Comment by wjcunningham7** [comment]
> @wjcunningham7 can this one be closed... since all infra related stuff has been moved out of core?

Yes

**Pull request closed without merging**